meta/run/update_mojang.py

```diff
@@ -61,7 +61,7 @@ def fetch_modified_version(path, version):
     }
 
     version_json["downloads"] = downloads
-    version_json["type"] = "old_snapshot"
+    version_json["type"] = "snapshot"
```

Comment:
I think you can actually just drop this line completely. From what I can tell, the jsons specify `snapshot` as their type already

Reply:
Not needed anymore: I trim the prefix for all now.